src/redux/actions/authActions.js

```diff
@@ -77,12 +77,26 @@ function logoutFailure() {
   };
 }
 
-function logoutSuccess() {
+/**
+ * This action has effects in multiple reducers. Since logout will
+ * affect every branch of the store by removing all of the information,
+ * it requires extra information to update those areas of the program.
+ * These extra pieces of information include empty arrays for the user,
+ * accoms, plans, trans, and trips (as well as fetching/synced booleans).
+ */
+// exported to allow testing
+export function logoutSuccess() {
   return {
     type: LOGOUT_SUCCESS,
     lastUpdated: Date.now(),
     isAuthenticated: false,
-    user: null
+    isFetching: false,
+    isSynced: true,
+    user: [],
+    accoms: [],
+    plans: [],
+    trans: [],
+    trips: []
   };
```

Comment:
This looks good. Though, ` isSynced: true` should be ` isSynced: false`.

Reply:
Good point! Changed it just now.